src/ActionUnit.h

```diff
@@ -77,7 +77,8 @@ class ActionUnit
     QList<TAction*> uninstallList;
 
 private:
-    ActionUnit() {}
+    ActionUnit() = default;
```

Comment:
This is a new C++11 feature for me, but Stack Exchange to the rescue with the slightly badly put: **[The new keyword `default` in C++11](https://stackoverflow.com/questions/20828907/the-new-keyword-default-in-c11#20828970)**.  I just wish I grokked it better...